### PR TITLE
fix: throw HTTP 400 on short password during signup

### LINKED ISSUE #5989: Wrong error throwing during sign up
**Describe the bug**
For short length of the password, the error is `HTTP 422`, It should be `HTTP 400`.

### PR DESCRIPTION
**Pull request by shreyanshdwivedi**
Fixes #5989 

#### Checklist

- [x] I have read the [Contribution & Best practices Guide](https://blog.fossasia.org/open-source-developer-guide-and-best-practices-at-fossasia) and my PR follows them.
- [x] My branch is up-to-date with the Upstream `development` branch.
- [x] The unit tests pass locally with my changes 
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] I have added necessary documentation (if appropriate)

- [ ] All the functions created/modified in this PR contain relevant docstrings.

#### Short description of what this resolves:
For short length of the password, the error is HTTP 422, It should be HTTP 400

#### Changes proposed in this pull request:
Adds a new exception - `BadRequest` to raise HTTP 400 error

### CODE REVIEW

**Comment by shreyanshdwivedi** [comment]
@mrsaicharan1 as we discussed, I've added a new exception because `return BadRequestError(..).respond` was not raising any error even when the `if(pass.length<8)` condition was satisfied. Do you think this is the correct approach @uds5501 @kushthedude ?
Once the implementation is finalized, I'll fix the hound and other checks.

**Comment by mrsaicharan1** [comment]
> @mrsaicharan1 as we discussed, I've added a new exception because `return BadRequestError(..).respond()` was not raising any error even when the `if(pass.length<8)` condition was satisfied. Do you think this is the correct approach @uds5501 @kushthedude ?
> Once the implementation is finalized, I'll fix the hound and other checks.

That's how you should do I believe because we need exceptions which are compliant with the  `JSONAPI` structure.

**Comment by mrsaicharan1** [comment]
@shreyanshdwivedi Just add a test here. Otherwise, LGTM
https://github.com/fossasia/open-event-server/blob/development/tests/all/integration/api/helpers/test_exceptions.py

**Comment by shreyanshdwivedi** [comment]
@mrsaicharan1 @uds5501 @kushthedude @iamareebjamal please review

File: app/api/admin_translations.py [hunk 1]
```diff
@@ -10,7 +10,6 @@
 translations_dir = 'app/translations'
 
 @admin_blueprint.route('/', methods=['GET'])
-@is_admin
```
Comment (kushthedude) [inline app/api/admin_translations.py]: Why did you remove the following decorator ?

Comment (shreyanshdwivedi) [reply app/api/admin_translations.py]: Ah! I was testing the translation thing on my local about which @mrsaicharan1 just opened the issue. Forgot to fix it. Fixing it now

**Comment by kushthedude** [comment]
@shreyanshdwivedi ?

**Comment by shreyanshdwivedi** [comment]
@kushthedude please refer to the issue. It is closed now
